migrations/20170304184220_add_constraints.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable quotes */
+
+exports.up = knex =>
+  knex.schema
+    .raw(`CREATE TYPE job_status AS ENUM ('pending', 'progress', 'complete', 'error')`)
```

Comment:
Yes we need, I want to be able to use Postico or psql without the risk of corrupting data.